### DISCUSSION TITLE
Status projekta

### DISCUSSION THREAD

**Discussion by zstapic** [foivz/pi21-vtucelj20-icuturic20-ipeles#3] [General]
Poštovane kolegice,

koji je trenutni status vašega projekta? Koji su vam planovi? Trebate li pomoć ili konzultacije?

Pogledao sam vašu dokumentaciju i ono što ste napravili do sada, pa vam dajem nekoliko smjernica za dalje:
- primijetim da ste u projektnoj dokumentaciji pokušali obraditi sve važne dijelove. Pojedina poglavlja imaju mogućnosti za poboljšanje, ali ništa strašno. Od prijedloga za poboljšanje izdvajam jaču povezanost projektnih aktivnosti i metodike razvoja uz napomenu da me raduje da imate knjigu kao osnovni izvor provođenja metodike. Također, predlažem jasniji prikaz (izrada je ok, ali prikaz može biti bolji) projektnog plana i gantograma. U cijenu koštanja bi bilo dobro uračunati i druge troškove i ukalkuliranu dobit, a ne samo cijenu rada. Ponuda koju ste izradili ne treba imati opisne dijelove niti radne sate. Opis može ići u obliku pisma uz ponudu. Bilo bi dobro kada bi ste imali definirane ključne pokazatelje uspjeha i praćenje istih kroz provedbu projekta. 
- poglavlje specifikacije korisničkih zahtjeva bi trebalo detaljnije raspisati, odnosno analizu korisničkih zahtjeva provesti sukladno standardu IEEE ili nekom drugom primjeru dobre prakse. Možete npr. u kontekstu funkcionalnih zahtjeva raspisati detaljne specifikacije za svaki slučaj korištenja, definirati setove podataka koji se koriste, napraviti skice ekrana. Čini se kako ste dio ovih artefakata smjestili u poglavlje o dizajnu, a zapravo bi dobro došli i u poglavlju specifikacije zahtjeva. U kontekstu nefunkcionalnih zahtjeva potrebno je svaki zahtjev koji ste naveli također detaljno raspisati uključujući i indikatore ispunjenja ovih zahtjeva.
- pri raspisu specifikacije dizajna proizvoda, slobodno kod svake funkcionalnosti dodajte i isječke modela podataka koji se na spomenutu funkcionalnost odnose. Dijagrami aktivnosti i slijeda koje ste prikazali imaju manje sintaksičke i semantičke nedostatke, a to možemo proći na konzultacijama. 
- bilo bi dobro identificirati i raspisati elemente krosfunkcionalnosti koji su zajednički svim funkcionalnostima
- bilo bi dobro koristiti više UML alata kod specifikacije dizajna, naravno, ovisno o tome gdje pojedini dijagram doprinosi razumijevanju teksta. 
- raduje me što ste počeli pisati programski kod
- do sada ostvareni bodovi 11.5

**Comment by vtucelj20** [comment]
Poštovani profesore, 

hvala Vam na povratnim informacijama.
Trenutni plan nam je projekt braniti na prvom roku 28.06.2021. 
Ukoliko imate vremena voljele bismo dogovoriti termin konzultacija na kojima bismo prošli kroz nedostatke koje ste naveli za dijagrame aktivnosti i slijeda, naš era model te mogućnost korištenja Entity Frameworka i LINQ-a za rad s bazom umjesto našeg prvog prijedloga (Database instance). 

Lijep pozdrav,
Valentina Tucelj

**Comment by zstapic** [comment]
Hvala vam za informacije. Obrana projekta u usklopu prvog ispitnog roka je
u redu.
Mogu li zamoliti za konzultacije sljedeći tjedan? Sada nam je stvarno gužva
oko kolokvija, evidencija, potpisa, obrana projekata.
Ako ste suglasni, molim vas javite se u utorak ili srijedu sljedeći tjedan
da se dogovorimo za termin. Ako vam je hitno, pokušat ću naći termin i ovaj
tjedan.
Hvala puno!

**Reply by vtucelj20** [reply]
Poštovani profesore,

nije hitno i odgovara nam sljedeći tjedan za konzultacije. Javit ćemo se u utorak ili srijedu za dogovor termina.

LP, 
Valentina Tucelj

**Reply by vtucelj20** [reply]
Poštovani profesore, 

javljamo se oko dogovora termina konzultacija kako smo se dogovorili prošli tjedan. Odgovara li Vam sutra(četvrtak) ili prekosutra(petak) neki termin za kratke konzultacije vezano uz nedostatke na dijagramima koje ste naveli i nekoliko naših pitanja?

LP, 
Valentina Tucelj

**Comment by zstapic** [comment]
Pozdrav,
hvala na podsjetniku. Predlažem četvrtak, 17.6. u 14:00 sati na
https://zoom.us/j/4572909028
Lp, Z.

**Comment by vtucelj20** [comment]
Termin je u redu.

Vidimo se, lp.

Valentina Tucelj

On 2021-06-16 12:01, Zlatko Stapić wrote:

> Pozdrav,
> hvala na podsjetniku. Predlažem četvrtak, 17.6. u 14:00 sati na
> https://zoom.us/j/4572909028
> Lp, Z.
> 
> On Wed, 16 Jun 2021 at 10:54, vtucelj20 ***@***.***> wrote:
> 
>> Poštovani profesore,
>> 
>> javljamo se oko dogovora termina konzultacija kako smo se dogovorili
>> prošli tjedan. Odgovara li Vam sutra(četvrtak) ili prekosutra(petak) 
>> neki
>> termin za kratke konzultacije vezano uz nedostatke na dijagramima koje 
>> ste
>> naveli i nekoliko naših pitanja?
>> 
>> LP,
>> Valentina Tucelj
>> 
>> --
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/foivz/pi21-vtucelj20-icuturic20-ipeles/discussions/3#discussioncomment-876976>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ABOHLZKOKZW3TIMQACZTFJ3TTBRGBANCNFSM46K62K2Q>
>> .
>> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].



Links:
------
[1] 
https://github.com/foivz/pi21-vtucelj20-icuturic20-ipeles/discussions/3#discussioncomment-877209
[2]
